From 67e40e38bdbb7488818bbe18da12995d59040a86 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@apple.com>
Date: Fri, 3 Jun 2011 02:34:28 -0400
Subject: [PATCH 11/16] XQuartz: Remove explicit link against libGL

This was indented to force a link against OpenGL.framework's libGL, but it
actually resulted in linking against mesa's libGL due to the ordering of -L

Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
---
 hw/xquartz/mach-startup/Makefile.am |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git xorg-server-1.10.99.901/hw/xquartz/mach-startup/Makefile.am xorg-server-1.10.99.901/hw/xquartz/mach-startup/Makefile.am
index 4b7b2bc..8e49ea8 100644
--- xorg-server-1.10.99.901/hw/xquartz/mach-startup/Makefile.am
+++ xorg-server-1.10.99.901/hw/xquartz/mach-startup/Makefile.am
@@ -39,7 +39,6 @@ X11_bin_LDADD += \
 	$(top_builddir)/glx/libglx.la
 
 X11_bin_LDFLAGS += \
-	-L/System/Library/Frameworks/OpenGL.framework/Libraries -lGL \
 	-Wl,-framework,OpenGL
 endif
 
-- 
1.7.5.2

